### DISCUSSION TITLE
package analytics question

### DISCUSSION THREAD

**Discussion by QarthO** [npm/feedback#949] [General]
Hey! I recently published a package. I wondered if there was an easy way to see how many downloads my package was getting. 
I noticed that for the first 24hrs, the package was live, the "weekly downloads" section wasn't even there. Since then it's been on one number. and hasn't moved. I've personally had the package downloaded so I know it should have gone up at least one.
<img width="410" alt="image" src="https://github.com/npm/feedback/assets/10179096/84789916-189c-4c79-9eed-f400585cd8a5">

My question is, how often is the weekly downloads number/graphs updated? 
After a week is there a way to track how many total downloads my package has gotten?
Downloads per version?

In general, is there a better tool to look at the analytics of a package?

**Comment by ljharb** [comment]
It usually goes up every day, and it's not ever been granular enough to be like an incrementing counter. Its value is in indicating relative popularity, not anything more - especially since most packages get downloaded by mirrors and private registries once and downloaded multiple times from there.
